101-vm-simpleWinServer/azuredeploy.json

```diff
@@ -0,0 +1,216 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vmName": {
+            "type": "string",
+            "defaultValue": "WinServerVM",
+            "metadata": {
+                "description": "The name of you Virtual Machine."
+            }
+        },
+        "VmSize": {
+            "type": "string",
+            "allowedValues": [
+                "Standard_D1_v2",
+                "Standard_D2_v2",
+                "Standard_D3_v2",
+                "Standard_D1_v2",
+                "Standard_D4_v2",
+                "Standard_D5_v2"      
+        ],
+            "metadata": {
+                "description": "The size of the VM"
+            },
+        "defaultValue": "Standard_D2_v2"
+        },
+        "adminUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "Username for the Virtual Machine."
+            }
+        },
+        "adminPassword": {
+            "type": "secureString",
+            "metadata": {
+                "description": "Password for the Virtual Machine. The password must be at least 12 characters long and have lower case, upper characters, digit and a special character (Regex match)"
+            }
+        },
+        "location": {
+        "type": "string",
+        "defaultValue": "[resourceGroup().location]",
+        "metadata": {
+                "description": "Location for all resources."
+            }
+        },
+        "windowsOSVersion": {
+            "type": "string",
+            "defaultValue": "2016-Datacenter",
+            "allowedValues": [
+                "2008-R2-SP1",
+                "2012-Datacenter",
+                "2012-R2-Datacenter",
+                "2016-Nano-Server",
+                "2016-Datacenter-with-Containers",
+                "2016-Datacenter",
+                "2019-Datacenter"
+            ],
+            "metadata": {
+                "description": "The Windows version for the VM. This will pick a fully patched image of this given Windows version."
+            }
+        }
+    },
+    "variables": {
+        "virtualNetworkName": "[concat('MyVNet-', parameters('vmName'))]",
```

Comment:
this isn't using the patter we were talking about in Teams - for the shared resources.

Also, remove "my" from the resource name and put the vmName first - this will sort the related resources so they are adjacent to one another